app/mailers/user_mailer.rb

```diff
@@ -120,7 +120,10 @@ def nsv_bsv_submission_rejected(submission:, artist:)
             unique_args: {
               submission_id: submission.id
             }
-    mail(to: submission.email, subject: 'An update about your submission')
+    title = submission.title ||= 'Unknown'
```

Comment:
Thank you very much @olerichter00 . You are correct `||` is enough. Infact my original intention was to assign 'Unknown' to the title so that the title matches the subject when we use submission.title in the email body. But with your suggestion I now realised I had already assigned @submission before even setting title. 🤦 . I will fix this in a new PR now.

Reply:
I guess it's fine and `||=` would work as well. But could lead to hard-to-find bugs.

Reply:
Yesss. True, It could cause bugs that will be extremely difficult to find. 🙌 . Will change now.

Reply:
thanks

Reply:
@olerichter00 Addressed here: https://github.com/artsy/convection/pull/1375 . Thank you!